build.gradle

```diff
@@ -243,6 +245,16 @@ task copyToRoot(type: Copy, dependsOn: [copyToLibs, jar, converterJar]) {
     rename "(.+)-$version(.+)", '$1$2'
 }
 
+task unixLineEndings( type: Exec) {
+    if(Os.isFamily(Os.FAMILY_WINDOWS)){
+        workingDir "$projectDir/code/sbin"
+        executable "bash"
+        args "-c", "fromdos $projectDir/code/pcgen.sh"
+    }else if(Os.isFamily(Os.FAMILY_UNIX)){
+        commandLine "dos2unix", "$projectDir/code/pcgen.sh"
+    }
+}
+
```

Comment:
lets hit that hurdle when we get to it. This version ties us to "bash" on unix which is highly likely to not be installed. On windows it solves a problem that only exists due to a misconfigured client.